es/docs/users.md

```diff
@@ -88,7 +88,7 @@ module Main
 end
 ```
 
-También puedes cambiar el mensaje por default en require_login, o puedes dejar el valor de nil si no quieres sacar ningún mensaje flash
+También puedes cambiar el mensaje por defecto en require_login, o puedes dejar el valor a nil si no quieres sacar ningún mensaje flash.
```

Comment:
perhaps 'dejar el valor nil' instead of 'dejar el valor de nil' would be better ?